ChameleonForms/FieldGenerators/Handlers/DateTimeHandler.cs

```diff
@@ -36,7 +37,15 @@ public override IHtmlString GenerateFieldHtml(IReadonlyFieldConfiguration fieldC
         public override void PrepareFieldConfiguration(IFieldConfiguration fieldConfiguration)
         {
             if (!string.IsNullOrEmpty(FieldGenerator.Metadata.DisplayFormatString))
-                fieldConfiguration.Attr("data-val-format", FieldGenerator.Metadata.DisplayFormatString.Replace("{0:", "").Replace("}", ""));
+            {

```

Comment:
I've got it, not so difficult. 

But I've found that a default value in DateTime field is dependable on CurrentCulture, not UICulture.
```
fieldGenerator.HtmlHelper.TextBoxFor(fieldGenerator.FieldProperty, fieldConfiguration.FormatString, attrs.ToDictionary())
```
Returns value formatted using CurrentCulture.
I have two options from here. Rewrite ChangeCulture to change both cultures or change implementation to use CurrentCulture instead CurrenUICulture. I'm going to read differences between them and why HtmlHelper uses CurrentCulture.